java/src/org/openqa/selenium/grid/data/NodeStatus.java

```diff
@@ -204,6 +204,14 @@ public long getLastSessionCreated() {
         .orElse(0);
   }
 
+  public String getBrowserVersion() {
+    return slots.parallelStream()
```

Comment:
No need for offloading to different threads here, this might add latency to spawn the threads.